src/app/item-popup/ItemSocketsWeapons.tsx

```diff
@@ -42,7 +42,11 @@ export default function ItemSocketsWeapons({
   }
 
   // Separate out perks from sockets.
-  const { intrinsicSocket, perks, modSocketsByCategory } = getWeaponSockets(item)!;
+  const { intrinsicSocket, perks, modSocketsByCategory } = getWeaponSockets(
+    item,
+    /* excludeEmptySockets */ false,
+    /* includeFakeMasterwork */ Boolean(item.crafted),
+  )!;
```

Comment:
This is why I generally prefer an options object instead of boolean parameters:

```suggestion
  const { intrinsicSocket, perks, modSocketsByCategory } = getWeaponSockets(
    item,
    { 
      excludeEmptySockets: false,
      includeFakeMasterwork: Boolean(item.crafted),
    }
  )!;
```

Better yet, you get to default them, so:

```suggestion
  const { intrinsicSocket, perks, modSocketsByCategory } = getWeaponSockets(
    item,
    { includeFakeMasterwork: Boolean(item.crafted) }
  )!;
```